middlewares/validators/user.js

```diff
@@ -172,6 +172,8 @@ async function validateUserQueryParams(req, res, next) {
           joi.array().items(joi.string().valid("IDLE", "OOO", "ACTIVE"))
         )
         .optional(),
+      role: joi.string().valid("in_discord", "member").optional(),
```

Comment:
Good catch randhir

Reply:
if I make role to array here I will have to handle the case of multiple entries in the array, which will go against the API contract and the requirements of this PR , so I think it would be better to do that when we implement search based on multiple roles